src/model/Item.php

```diff
@@ -53,7 +53,7 @@ public function toRSS(DOMElement $elem_item)
         $elem_item->appendChild($xml->createElement("description"))->appendChild($fragment);
 
         $elem_item->appendChild($xml->createElement("author", htmlspecialchars($this->getAuthorWithEmail())));
```

Comment:
We should try here:
```
$elem_item->appendChild($xml->createElementNS ("http://purl.org/dc/elements/1.1/", "dc:creator", htmlspecialchars($this->author)));
```
Is this working and validating aggainst https://validator.w3.org/feed/ ?